src/igvfd/schemas/biomarker.json

```diff
@@ -81,9 +81,10 @@
     },
     "fuzzy_searchable_fields": [
         "@type",
-        "name",
         "gene",
-        "quantification"
+        "name",
+        "quantification",
```

Comment:
searching quantification means searching an item like "positive"  Searching the summary means searching CD38 positive.  I think we should search the summary